app/assets/javascripts/models/data/service_options.js.coffee

```diff
@@ -119,7 +118,12 @@ ns.ServiceOptions = do (ko, edsc = @edsc, KnockoutModel = @edsc.models.KnockoutM
                 if !accessMethodMatched && matches[1] == jsonObj.collection_id && parseInt(matches[2], 10) == index
                   accessMethodMatched = true
                   @loadForm(true) if jsonObj.type == 'service'
-                  ko.applyBindingsToNode(echoformContainer, {echoform: this})
+                  # After the binding is complete, toggle select elements to trigger validation lest they be pruned
+                  $.when(ko.applyBindingsToNode(echoformContainer, echoform: this)).done ->
```

Comment:
This only works for "North Polar Stereographic". It doesn't even work for South Polar Stereographic...

See my earlier inline comments in knockout_extensions.js.coffee.

The following is a chunk of code that you can put within $.when...done block. It is just a general idea, I didn't test it and it may even have syntax errors:
```
projectionOptions = $("[id$=-projectionSettings]").filter(":visible")
if projectionOptions.length > 0
    reprojectionOptionsLabel = $('label:contains("Re-projection Options"):visible')
    if reprojectionOptionsLabel.length > 0
        # touch all select dropdowns
        reprojectionSelect = $('#' + reprojectionOptionsLabel.attr('for'))
        originalVal = reprojectionSelect.val()
        reprojectionSelect.val('&').change().val(originalVal).change() # change() is better than trigger('change') here, I think
        # touch all inputs
        inputs = projectionOptions.find('input[type=text]:visible')
        $.each(inputs, (i, obj) ->
            originalVal = $(obj).val()
            $(obj).val(originalVal + 1).change().val(originalVal).change()
        )


```